apps/nextjs-website/react-components/types/HighlightBox/HighlightBox.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface HighlightBoxProps {
+  title: string;
+  body: string;
+  imageURL: string;
+  eyelet?: string | null;
```

Comment:
```suggestion
  eyelet?: string;
```
Handling undefined and null is redundant.

Do not accept null as a value. If a value is null, it should not be passed to the component at all.